desktop/client-sdk/build/index.d.ts

```diff
@@ -0,0 +1,103 @@
+type AppConstructorParam = {
```

Comment:
Add `.gitignore` to prevent push build output files to vsc.

Reply:
> Add `.gitignore` to prevent push build output files to vsc.

Npm publish will ignore file in .gitignore. I refer to the axios project, they also push the build files to vsc.

Reply:
OK. Understand.